RealmSwift-swift2.0/Tests/RealmCollectionTypeTests.swift

```diff
@@ -101,7 +101,9 @@ class RealmCollectionTypeTests: TestCase {
     }
 
     func testDescription() {
+        // swiftlint:disable line_length
```

Comment:
Filed an issue on SwiftLint for this: https://github.com/realm/SwiftLint/issues/222. If you have any ideas on what the syntax would look like, please share.